### ISSUE TITLE
where to change port 8080

### ISSUE THREAD

**Issue by mcwago** [braiins/farm-proxy#12]
ERROR farm_proxy::http_api: Cannot bind address for monitoring 0.0.0.0:8080
2023-01-07T18:26:01.985816Z ERROR warp::server: error binding to 0.0.0.0:8080: error creating server listener: Address already in use (os error 98)
2023-01-07T18:26:01.985866Z  INFO farm_proxy::http_api: Monitoring running on 0.0.0.0:8080

**Comment by praeluceo** [comment]
I came here to open this same ticket. farm_proxy depends on port 8080, but so does LND API. I want to get farm-proxy working as an umbrel app, but that means it can't use standard ports (lnd shouldn't have used 8080 either) but the FP http_api needs to use a different port or make it straight-forward to change the port.

I did try changing the port in prometheus here:
~/farm-proxy$ grep -R 8080
monitoring/prometheus/prometheus.yml:      - targets: ["localhost:8080"]

However, that did not change the port that the http_api was actually listening on. Can instructions be provided on how to modify the ports?

Also, for inclusion in umbrel, editing the pool list will need to be able to be performed from the web dashboard:
https://github.com/getumbrel/umbrel-apps
But for farm-proxy step #1 is already completed!
